unit-tests-jest/numbers.test.ts

```diff
@@ -0,0 +1,86 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
+describe('evenNumbers', () => {
+  it('returns the even numbers', () => {
+    const numbers = [1, 4, 5, 10, 0]; // Arrange
+    const result = evenNumbers(numbers); // Act
+    expect(result).toEqual([4, 10, 0]); // Assert
+  });
+
+  it('returns empty', () => {
+    const numbers = [1, 3, 5, 7, 9];
+    const result = evenNumbers(numbers);
+    expect(result).toEqual([]);
+  });
+});
+
+describe('toDollars', () => {
+  it('returns integer converted to dollars', () => {
+    const number = 84;
+    const result = toDollars(number);
+    expect(result).toEqual('$84.00');
+  });
+
+  it('returns float round-up converted to dollars', () => {
+    const number = 34.99954;
+    const result = toDollars(number);
+    expect(result).toEqual('$35.00');
+  });
+
+  it('returns float round-down converted to dollars', () => {
+    const number = 46.23178;
+    const result = toDollars(number);
+    expect(result).toEqual('$46.23');
+  });
+
+  it('returns float one decimal converted to dollars', () => {
```

Comment:
I like this test. I didn't think of doing it myself, but it's a great edge case to test!